src/libstd/io/mod.rs

```diff
@@ -620,6 +620,16 @@ pub trait BufRead: Read {
     fn lines(self) -> Lines<Self> where Self: Sized {
         Lines { buf: self }
     }
+
+    /// Returns an iterator over the lines of this reader, separated by either `\n` or `\r\n`.
+    ///
+    /// The iterator returned from this function will yield instances of
+    /// `io::Result<String>`. Each string returned will *not* have the newline
+    /// separator at the end.
+    #[unstable(feature = "io", reason = "Just recently added.")]
```

Comment:
We avoid blanket feature names nowadays, so can you rename this feature to `lines_any`?